### PR TITLE
#544 | update docs for code converter not found on windows

### PR DESCRIPTION
**Pull request by hoavt**
connect #544 

@phuonglm please review.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -152,7 +152,7 @@ Automatic Installation on Windows
 Manual Installation on Windows
 ------------------------------
 
-This should be the same on Windows 10, Windows 8 and Windows 7.
+This should be the same on Windows 10, Windows 8 and Windows 7 SP1 x64. We don't support Windows 32 bit.
```
Comment (phuonglm) [inline docs/getting_started.rst]: this should add at 
```
We recommend the following tested and supported platforms:

macOS Sierra
Ubuntu 16.04
Windows 10, Windows 8, Windows 7
```

File: docs/getting_started.rst [hunk 2]
```diff
@@ -333,6 +333,31 @@ teracy-dev Git Clone and Vagrant Up
...
          has an installation issue which was reported `here <https://www.virtualbox.org/ticket/4140>`_
 
+       - On Windows, if you see the error `code converter not found (UTF-16LE to Windows-1258) (Encoding::ConverterNotFoundError)` when using vagrant:
+
+        ::
+
+          $ vagrant status
+          [TeracyDev::Util][WARN]: The _id: '0' is deprecated, use the _id: 'kernel-core' instead for {"_id"=>"0", "location"=>{"git"=>{"remote"=>{"origin"=>"https://github.com/teracyhq-incubator/teracy-dev-core.git"}, "branch"=>"develop"}}, "require_version"=>">= 0.4.0-SNAPSHOT"}
+
+          [TeracyDev::Plugin][INFO]: installing plugin: {"_id"=>"essential-0", "name"=>"vagrant-hostmanager", "version"=>"", "state"=>"installed", "config_key"=>"hostmanager", "enabled"=>true, "options"=>{"enabled"=>true, "manage_host"=>true, "manage_guest"=>true}, "sources"=>["https://rubygems.org/", "https://gems.hashicorp.com/"]}
```
Comment (datphan) [inline docs/getting_started.rst]: we should keep relevant message, remove `The _id: '0' is deprecated ...`, remove this line `installing plugin:...`

File: docs/getting_started.rst [hunk 3]
```diff
@@ -333,6 +333,31 @@ teracy-dev Git Clone and Vagrant Up
...
+
+          C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:185:in `encode!': code converter not found (UTF-16LE to Windows-1258) (Encoding::ConverterNotFoundError)
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:185:in `initialize'
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:250:in `exception'
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:250:in `raise'
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:250:in `check'
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:320:in `QueryValue'
+          from C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:644:in `read'
```
Comment (datphan) [inline docs/getting_started.rst]: add `...` at bottom

File: docs/getting_started.rst [hunk 4]
```diff
@@ -333,6 +333,31 @@ teracy-dev Git Clone and Vagrant Up
...
+
+
+        you should set the `system locale` into `US`, by going to `Control Panel` --> `Region`
```
Comment (datphan) [inline docs/getting_started.rst]: shift tab these line for better display

Comment (phuonglm) [inline docs/getting_started.rst]: not in part not in quote and the error message log could just 
```
$ vagrant status
C:/HashiCorp/Vagrant/embedded/mingw64/lib/ruby/2.4.0/win32/registry.rb:185:in `encode!': code converter not found (UTF-16LE to Windows-1258) (Encoding::ConverterNotFoundError)
```

**Review by phuonglm — CHANGES_REQUESTED** [review]

**Comment by hoavt** [comment]
Applied your comments.

**Comment by phuonglm** [comment]
LGTM

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**